ios/Video/RCTVideo.swift

```diff
@@ -215,6 +216,42 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     }
     
     // MARK: - Player and source
+
+    @objc
+    func setUpPlayer(_ playerItem:AVPlayerItem!) {
+        _playerObserver.player = nil
+        _playerObserver.playerItem = nil
+
+        if #available(iOS 10.0, *) {
+            self._player = AVQueuePlayer(playerItem: playerItem)
```

Comment:
Don't we need the same check here to keep the old player if it's available like in line 234?
```suggestion
            self._player = self._player ?? AVQueuePlayer(playerItem: playerItem)
```